packages/client/src/flow/sendCollectionToTreasury.tx.js

```diff
@@ -8,11 +8,6 @@ export const SEND_COLLECTION_TO_TREASURY = `
 
 		prepare(signer: AuthAccount) {
 	
-			// borrow a reference to the signer's NFT collection
```

Comment:
just curious on the reason for removing this part

Reply:
It's not used anywhere?

Reply:
Got it, I thought it was pre check for the collection's existence.

Reply:
It maybe was before, but now it seems the user's collection is not relevant anymore because it's ignored and a new collection is created